frontend/components/context/Notifications/Notification.tsx

```diff
@@ -0,0 +1,38 @@
+import { faXmarkCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classnames from "classnames";
+
+import { Notification as NotificationType } from "./NotificationProvider";
+
+interface NotificationProps {
+  notification: NotificationType;
+  clearNotification: (text?: string) => void;
+}
+
+const Notification = ({
```

Comment:
If required, `Notification` can be outfitted with a `timeout` capability that calls the `clearNotification` after a specified timeout. I didn't add that since I didn't believe it to be within scope.

Reply:
I think the timeout capability is really great is that something that you would be interested in doing or no? I actually had some ideas about that, will add them in a separate comment now.

Reply:
Do you want to list them as a separate issue? Might be worth having all those requirements localized to notification improvements. 

I'm open to working on whatever you have in mind.

Reply:
Yes, sounds good! Thank you so much. Let's merge this one then. And I will put some ideas I had in a separate issue :)